### PR TITLE
Add document get

### PR DESCRIPTION
**Pull request by Aschen**
## What does this PR do?

Add command to get a document

```
USAGE
  $ kourou document:get INDEX COLLECTION ID

ARGUMENTS
  INDEX       Index name
  COLLECTION  Collection name
  ID          Document ID

OPTIONS
  -h, --host=host      [default: localhost] Kuzzle server host
  -p, --port=port      [default: 7512] Kuzzle server port
  --help               show CLI help
  --password=password  Kuzzle user password
  --ssl                Use SSL to connect to Kuzzle
  --username=username  [default: anonymous] Kuzzle user
```

### How should this be manually tested?

  - Step 1 :
  - Step 2 :
  - Step 3 :  
  ...

### CODE REVIEW

File: src/commands/instance/logs.ts [hunk 1]
```diff
@@ -34,11 +33,9 @@ export class InstanceLogs extends Command {
 
     try {
       await this.showInstanceLogs(instance, followOption)
-    }
-    catch {
+    } catch {
```
Comment (Yoann-Abbes) [inline src/commands/instance/logs.ts]: You shouldnt change it
Same for others

File: src/commands/instance/logs.ts [hunk 2]
```diff
@@ -58,24 +55,22 @@ export class InstanceLogs extends Command {
...
 
     const containersList: string[] = containersListProcess.stdout.replace(/"/g, '').split('\n')
 
     return containersList.filter(containerName => {
-      if ( containerName.includes('kuzzle')
-        && !containerName.includes('redis')
-        && !containerName.includes('elasticsearch')
+      if (containerName.includes('kuzzle') &&
+        !containerName.includes('redis') &&
+        !containerName.includes('elasticsearch')
```
Comment (Yoann-Abbes) [inline src/commands/instance/logs.ts]: You shouldnt change that

File: src/commands/instance/logs.ts (hunk 1 above)
Comment (Aschen) [reply src/commands/instance/logs.ts]: Thats not me, thats prettier :expressionless: 
He is activated on the project and auto indent the code

File: src/commands/instance/logs.ts (hunk 2 above)
Comment (Aschen) [reply src/commands/instance/logs.ts]: Prettier -_-

File: src/commands/instance/logs.ts [hunk 3]
```diff
@@ -34,48 +33,41 @@ export class InstanceLogs extends Command {
...
-        && !containerName.includes('elasticsearch')
-      ) {
-        return true
-      }
-      return false
-    });
+    return containersList.filter(containerName =>
+      (containerName.includes('kuzzle') &&
+        !containerName.includes('redis') &&
+        !containerName.includes('elasticsearch')))
```
Comment (Yoann-Abbes) [inline src/commands/instance/logs.ts]: Prettier ? :trollface:

**Pull request merged**